.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature Request
+description: Suggest a new feature
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **Attention:** if this is a _question_ about how to build / test / query / deploy using
+        Bazel intellij, or a _discussion starter_, send it to bazel-discuss@googlegroups.com or discuss on
+        [Bazel Slack](https://slack.bazel.build/) #intellij channel instead!
+  - type: textarea
+    id: desc
+    attributes:
+      label: >
+        Description of the feature request:
+  - type: textarea
+    id: problem
+    attributes:
+      label: >
+        What underlying problem are you trying to solve with this feature?
+  - type: input
+    id: os
+    attributes:
+      label: >
+        What operating system, Intellij IDE and programming languages are you using? Please provide specific versions.
+  - type: textarea
+    id: relevant-info
+    attributes:
+      label: Have you found anything relevant by searching the web?
+      description: >
+        Places to look:
+
+         * GitHub issues: https://github.com/bazelbuild/intellij/issues
+         * Email threads on https://groups.google.com/forum/#!forum/bazel-discuss
```

Comment:
We should also update this to https://groups.google.com/forum/#!forum/intellij-bazel-plugin